sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaUnboundedReader.java

```diff
@@ -509,6 +521,13 @@ String name() {
 
     List<TopicPartition> partitions =
         Preconditions.checkArgumentNotNull(source.getSpec().getTopicPartitions());
+
+    // Each source has a single unique topic.
```

Comment:
Is this true? I would think that its possible for a source to consume from multiple topics, depending on how the work was split

Reply:
For the DF implementation, we create separate readers for each topic though, so for this case, we'll still get correct metrics.

Reply:
SG, thanks